quiz-participant-client/src/quiz/QuizDashboard.tsx

```diff
@@ -54,6 +57,15 @@ const QuizDashboard: React.FC<QuizDashboardProps> = (props: QuizDashboardProps)
                 headers: {
                     'Content-Type': 'text/plain',
                 }
+            })
+            .then(response => {
+                if (response.status > 400) {
```

Comment:
Funktioniert das tatsächlich bei dir? Wenn ich eine falsche Quiz ID eingebe, bekomme ich erwartungskonformerweise einen 400 Status zurück und dann wird diese Bedinung nicht wahr.

Reply:
Ok, nachdem ich die Container ohne Cache nochmal neugebaut habe, ist das Verhalten unverändert. Aber ich glaube, ich habe das Problem nun verstanden. Anscheinend muss die Quiz-ID, die man eingibt gewissen Kriterien erfüllen, GUID Format, Mindestlänge, keine Ahnung... Wenn ich eine reguläre Quiz ID eingebe, die nur lokal nicht existiert, bekomme ich einen 404. Wenn ich aber einfach Quatsch eingebe oder eine reguläre GUID kürze, dann kommt der 400. 

27ba42f6-cf9c-4556-b57f-37b9e15f5483 => 404
27ba42f6-cf9c-4556-b57f => 400

Wenn es dafür gute Grunde gibt, wäre es sinnvoll für den 400 auch eine entsprechende Fehlermeldung anzuzeigen. "Die eingegebene Quiz ID ist keine gültige ID" oder sowas.

Reply:
Ja, Der 404er kommt von mir, der 400er vom Framework wenn der String nicht in eine UUID deserialisiert werden kann. Ich fange den 400er jetzt auch im Frontend ab.